deployment/mqtt-to-blob/Dockerfile

```diff
@@ -7,6 +7,9 @@ WORKDIR /app
 # Install any needed packages specified in requirements.txt
 COPY ./mqtt-to-blob/requirements.txt ./
 RUN pip install --trusted-host pypi.python.org -r requirements.txt --no-cache-dir
+RUN pip3 install numpy
```

Comment:
Please use the requirements.txt for this. And there it is also used, so I recommend removing this.